traffic_ops/app/db/create_tables.sql

```diff
@@ -841,7 +835,6 @@ CREATE TABLE IF NOT EXISTS deliveryservice_tmuser (
     CONSTRAINT idx_89525_primary PRIMARY KEY (deliveryservice, tm_user_id)
 );
 
-
 ALTER TABLE deliveryservice_tmuser OWNER TO traffic_ops;
```

Comment:
The `deliveryservice_tmuser` table was removed in `2022031612431414_remove_unused_ds_tmuser_table.up.sql`:

https://github.com/apache/trafficcontrol/blob/2257ddb1a49c0b157206a92e753a01ce46eab165/traffic_ops/app/db/migrations/2022031612431414_remove_unused_ds_tmuser_table.up.sql#L18